swagger_spec_validator/common.py

```diff
@@ -30,10 +36,18 @@ def validate_json(json_document, schema_path):
     schema_path = resource_filename('swagger_spec_validator', schema_path)
     with open(schema_path) as schema_file:
         schema = json.loads(schema_file.read())
-    resolver = RefResolver('file://{0}'.format(schema_path), schema)
+    resolver = RefResolver(
+        'file://{0}'.format(schema_path),
+        schema,
+        store={'http://swagger.io/v2/schema.json': schema})
```

Comment:
Oops, this should be `store=schema['id']`

I believe it should avoid a network call. The way refs are looked up, they use `urljoin(current_scope, json_pointer)` where json_pointer is some `#/path/to/` ref.  If anything uses the absolute path to the schema, it would cause a network call.

I guess this might not happen very often in practice. It might have been triggered by the bug that is being fixed in `jsonschema` under python 3.